lib/dialects/postgres/query/pg-querycompiler.js

```diff
@@ -175,19 +176,26 @@ class QueryCompiler_PG extends QueryCompiler {
   columnInfo() {
     const column = this.single.columnInfo;
     let schema = this.single.schema;
+    let table = this.single.table;
+
+    // Ref can define the tableName and schema
+    if (table instanceof Ref) {
+      schema = table._schema;
+      table = table.ref;
+    }
 
     // The user may have specified a custom wrapIdentifier function in the config. We
     // need to run the identifiers through that function, but not format them as
     // identifiers otherwise.
-    const table = this.client.customWrapIdentifier(this.single.table, identity);
+    table = this.client.customWrapIdentifier(table, identity);
 
     if (schema) {
       schema = this.client.customWrapIdentifier(schema, identity);
     }
 
     let sql =
-      'select * from information_schema.columns where table_name = ? and table_catalog = current_database()';
-    const bindings = [table];
+      'select * from information_schema.columns where table_name = ? and table_catalog = ?';
+    const bindings = [`${table}`, this.client.database()];
```

Comment:
is this change actually related to this PR?

Reply:
This was a conscious change: https://github.com/knex/knex/pull/4623/files

Reply:
Probably a result of the rebase. Will switch it back